### DISCUSSION TITLE
Tradeseq for scATAC seq data

### DISCUSSION THREAD

**Discussion by Giane98** [statOmics/tradeSeq#207] [Q&A]
Hi guys,

First of all, I would really like to thank you very much for the package and your work, I’ve seen it appear in a Nature paper a month ago and thought it was very cool and useful, so now I am trying to apply it to my datasets as well. I am working on a scRNA-ATAC seq multiomic dataset and I already tried out the package for the scRNA seq data. However, I would also like to try out modelling the differentially accessible regions along a trajectory, but I cannot find a vignette explaining the application of traseq to scATAC seq data, do you know if such a vignette exists or if there are any papers applying tradeseq to scATAC data?

Thank you very much!

**Comment by PeiyuShii** [comment]
Hi! I am also interested in applying tradeseq to scATAC data. Have you found any vignettes explaining the application of traseq to scATAC seq data?
